requirements-test.txt

```diff
@@ -1,8 +1,6 @@
-pytest==6.2.5
-scikit-learn==1.0.2 ; python_version >= '3.7'
-scikit-learn==0.24 ; python_version == '3.6'
-pandas==1.1.5 ; python_version <= '3.6'
-pandas==1.2.2 ; python_version >= '3.7' and python_version < "3.10"
-pandas==1.4.0 ; python_version >= '3.10'
+pytest
 numpy>=1.19.5 ; python_version <= '3.9'
-numpy>=1.21.6 ; python_version >= '3.10'
+numpy>=1.21.6 ; python_version == '3.10'
+numpy>=1.23.5 ; python_version == '3.11'
+scikit-learn
```

Comment:
this would lead to non reproducible env. It's worth sticking to version . and just get it updated